report-pr-errors

```diff
@@ -21,7 +21,7 @@ from alibot_helpers.github_utilities import (
     calculateMessageHash, setGithubStatus, parseGithubRef, GithubCachedClient,
     PickledCache,
 )
-from alibot_helpers.utilities import to_unicode
+from alibot_helpers.utilities import to_unicode, getout
```

Comment:
maybe a better approach would be to expose the function in alibuild to retrieve the version?

Reply:
I'm not sure I get what you mean. Currently in ali-bot we only pickup whatever alibuild we find in $PATH, you'd like to import it as a python module instead?